qforce/fragment.py

```diff
@@ -32,13 +32,16 @@ def fragment(mol, qm, job, config):
 
         if name not in unique_dihedrals:
             unique_dihedrals[name] = term.atomids
-
+    generated = [] # Number of fragments generated but not computed
     for name, atomids in unique_dihedrals.items():
         frag = Fragment(job, config, mol, qm, atomids, name)
         if frag.has_data:
             fragments.append(frag)
+        if frag.has_inp:
+            generated.append(frag)
 
-    check_and_notify(job, config.scan, len(unique_dihedrals), len(fragments))
+    check_and_notify(job, config.scan, len(unique_dihedrals), len(fragments),
+                     len(generated))
```

Comment:
Not too important but: I use line length of max 100, would be good to be consistent with that. (might apply elsewhere also)